### DISCUSSION TITLE
v6.2.4

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#493] [公告]
> 🥮本版本由青红丝五仁月饼赞助发布，豆沙月饼提供技术支持

## 新功能

* 新增对 macOS(.dmg)、Debian/Ubuntu(.deb)、RedHat/Fedora(.rpm)、Synology DSM 7.x(.spk) 的软件包打包（实验性） @Ghost-chu  @Gaojianli 
  * 其中，群晖 DSM 的软件包使用 Docker 实现，安装套件时，将自动配置和管理 Docker 容器和路径挂载
  * Deb 安装包需要 `java-runtime(>=21)`
* 添加对 qBittorrent 4.x 的 Private Torrent 私有种子跳过支持 @ccloli @paulzzh 

## 中国大陆地区网络定向优化

* 新增 IPDB 中国大陆 CDN 备用源。当下载失败或未在 120 秒内完成下载时，将自动回退使用备用源。
* 新用户默认使用 `sparkle.ghostchu.com/ping/config` 连接 BTN 服务器获取配置
  * 具体提交和后续获取规则使用的服务器，由 BTN 服务器根据您的 IP 地址动态下发配置

## 用户体验改进

* 当鼠标悬浮在 WebUI 的报错消息上时，消息不再自行消失 @MirrorCY 
* 新增 IPDB 境内 CDN 备用更新源。如果 PBH 未在 2 分钟内完成文件下载/或下载失败，PBH 将自动切换到备用更新源重试 @Ghost-chu 
* 新安装的用户现在 BTN 服务器默认使用境内 CDN 配置端点 @Ghost-chu 
  * BTN 服务器将根据用户 IP 所属国家/地区下发不同的配置文件
* 当暂无趋势数据时，显示空占位符 @Gaojianli 

## 性能优化

* 封禁计数图表（按日）查询优化，提升约 80% 的查询速度 @Ghost-chu 

## 错误修复

* 修复了解决依赖项的镜像服务器测试时可能无限期卡住的问题 @Ghost-chu 
* 修复 WebUI 页脚的一个显示错误 @Gaojianli 
* 修复程序使用的图标不是正方形，导致的显示问题 @Ghost-chu 
* 修复本地即使有 IPDB 文件，但 IPDB 文件需要更新且更新失败时，IPDB 被错误禁用的问题 @Ghost-chu 
* 修复 PeerRecord 保存失败时，会无限重试，导致内存缓存队列上涨耗尽堆内存的问题 @Ghost-chu 
* 修复 IP 地址无效时，仍然会被处理的问题 @Ghost-chu 
* 过滤 qBittorrent 上的 Tor 和 I2P 用户地址，避免处理出错 @Ghost-chu 

### Vuze/Azureus

* 合并 BiglyBT 适配器的更改

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.2.4'>v6.2.4</a>.</em>

**Comment by Mishasama** [comment]
请问赞助商的具体品牌是什么？这样好像没什么广告效果啊……赞助商会不高兴吧？🤔
